pom.xml

```diff
@@ -119,4 +127,4 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
-</project>
+</project>
```

Comment:
Please add a newline at the end of the file.

Reply:
done